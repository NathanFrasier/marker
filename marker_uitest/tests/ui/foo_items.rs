mod good {
    fn foo() {}

    static FOO: i32 = 0;

    struct Foo {
        foo: i32,
    }
}

mod foo {
    use crate::good as foo;

    const FOO: i32 = 0;

    enum Foo {
        Foo,
    }
}

fn main() {}
