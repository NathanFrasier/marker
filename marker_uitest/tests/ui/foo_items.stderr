warning: found a `module` item with a test name
  --> $DIR/foo_items.rs:1:1
   |
1  | / mod find_me {
2  | |     const FIND_ME_CONST: i32 = 0;
3  | |
4  | |     static FIND_ME_STATIC: i32 = 0;
...  |
18 | |     trait FindMeTrait {}
19 | | }
   | |_^
   |
   = note: `#[warn(marker::item_with_test_name)]` on by default

warning: found a field with a test name
 --> $DIR/foo_items.rs:9:9
  |
9 |         find_me_field: i32,
  |         ^^^^^^^^^^^^^^^^^^

warning: found an enum variant with a test name
  --> $DIR/foo_items.rs:12:9
   |
12 |         FindMe,
   |         ^^^^^^

warning: found a `use` item with a test name
  --> $DIR/foo_items.rs:21:1
   |
21 | use find_me::find_me_fn as find_me;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: 4 warnings emitted

